e2e/test/scenarios/dashboard-filters/reproductions/24500-gracefully-deal-with-corrupted-filter.cy.spec.js

```diff
@@ -119,11 +119,16 @@ describe("issues 15279 and 24500", () => {
     saveDashboard();
 
     // Check the list filter again
-    filterWidget().contains("List").click();
+    filterWidget().contains("List").parent().click();
     cy.wait("@values");
 
-    // Check that the search filter works
+    cy.log("Check that the search filter works");
+
+    // reset filter value
+    filterWidget().contains("Search").parent().icon("close").click();
```

Comment:
when filterWidget contains some value, title `Search` still presents in DOM, but it's not a root element anymore